Framework/Crystal/src/RotateSampleShape.cpp

```diff
@@ -0,0 +1,179 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
```suggestion
// Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
```
I'm always forgetting to update this as well!

Reply:
Thanks for noting this!! I could have used [class_maker.py](https://developer.mantidproject.org/ToolsOverview.html#class-maker-py) to generate this file.